packages/component-library-vue/src/CustomRadioButton.vue

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+  import { defineComponent } from "vue";
+  import { useModelWrapper } from "./helpers/modelWrapper";
+
+  export default defineComponent({
+    name: "UtrechtCustomRadioButton",
+    props: {
+      invalid: { type: Boolean, required: false },
+      modelValue: { type: [String, Number, Boolean], require: false, default: "" },
+    },
+    setup(props, { emit }) {
+      return {
+        value: useModelWrapper(props, emit),
+      };
+    },
+  });
+</script>
+
+<template>
+  <input
+    v-model="value"
+    type="radio"
+    :aria-invalid="invalid || undefined"
+    :class="[
+        { 'utrecht-custom-radio-button--invalid': invalid },
+        'utrecht-custom-radio-button',
+        'utrecht-custom-radio-button--html-input'
```

Comment:
For consistency I do kinda wish we had a `utrecht-custom-radio-button--html-input` (to separate out the pseudo-class selectors), but we don't have one implemented yet.

I suggest we either implement `utrecht-custom-radio-button--html-input` or remove this class name for now.